ivy/functional/backends/jax/experimental/linear_algebra.py

```diff
@@ -148,3 +148,15 @@ def cond(
     out: Optional[JaxArray] = None,
 ) -> JaxArray:
     return jnp.linalg.cond(x, p=p)
+
+
+def solve_triangular(
+    a: JaxArray,
+    b: JaxArray,
+    /,
+    *,
+    lower: bool = True,
+    unit_diagonal: bool = False,
```

Comment:
You are also returning `transpose` from the `Ivy API`. How are handling it here ??